triton-client/src/store/store.ts

```diff
@@ -37,14 +37,15 @@ export type AppAction =
     | RunsStateAction
     | ConstantsStateAction
 
-const loggerMiddleware: [Middleware<{}, RootState>] | [] =
-    process.env.NODE_ENV === "development"
-        ? [
-              createLogger({
-                  level: "info",
-              }),
-          ]
-        : []
+// below logger isnt being used
```

Comment:
delete this if this is useless...

Reply:
It's kinda useful for logging redux actions in the console. I personally only use the Redux extension or manually log to console.